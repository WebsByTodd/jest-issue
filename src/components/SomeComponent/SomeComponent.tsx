import * as React from "react";
const html = require("./index.html");

class SomeComponent extends React.Component<{}, {}> {
  constructor(props: {}) {
    super(props);
  }
  render() {
		return (
    	<p dangerouslySetInnerHTML={html} />
		);
  }
}

export default SomeComponent;
