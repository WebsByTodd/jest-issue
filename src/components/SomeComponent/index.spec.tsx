import * as React from "react";
import * as renderer from "react-test-renderer";

import SomeComponent from "./SomeComponent";

test("SomeComponent renders", () => {
	const tree = renderer.create(<SomeComponent />).toJSON();
	expect(tree).toMatchSnapshot();
});
