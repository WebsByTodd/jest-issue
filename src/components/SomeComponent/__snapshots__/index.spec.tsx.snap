// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SomeComponent renders 1`] = `
<p
  dangerouslySetInnerHTML="<p>Some Html</p>
"
/>
`;
